docs/smt_and_rmt.md

```diff
@@ -41,7 +41,7 @@ From SUSE Linux Enterprise Server 15 onward, only RMT or SUMA is available.
 |Clean up data from repositories that are not used any longer|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
 |Bash completion|:x:|:heavy_check_mark:|:x:|
 |Available on [openSUSE Leap 15](https://github.com/SUSE/rmt/blob/master/docs/installation.md#installation-on-opensuse-leap-15)|:x:|:heavy_check_mark:<sup>[2](#self-support)</sup>|:heavy_check_mark: (Uyuni, #self-support)|
-|Option to [run as container](https://github.com/SUSE/rmt/blob/master/README.md#development-setup---docker-compose)|:x:|:heavy_check_mark:<sup>[2](#self-support)</sup>|:x:|
```

Comment:
Dumb question, but has the k8s feature replaced docker?

Reply:
K8s is for production deployments, whereas docker-compose is for development purposes.
I think for the purposes of this table, it's more relevant to show the k8s option.